addons/web_editor/static/lib/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -31,6 +31,10 @@ import {
 Text.prototype.oDeleteBackward = function (offset, alreadyMoved = false) {
     const parentNode = this.parentNode;
 
+    if(parentNode && parentNode.classlist && parentNode.classlist.include('css_editable_mode_display')){
+        return;
+    }
+
```

Comment:
This doesn't look to be the right fix because it targets a special class of `website` but does so in `web_editor`. I've had a look at the issue on a runbot and I found that all those nodes in the video thing are actually inside a `contenteditable="false"`. Therefore, our commands should do nothing in there. We seem to have overlooked the case where the selection is in a non-editable area. I tried setting a random `P` element `contenteditable="false"` and I could reproduce the issue there as well, that is I could delete some content of the node even though it should not be editable.

Reply:
@dmo-odoo  , I updated the fix , let me know what you think , Thanks !
Note:  The runbot should be green, I just commited again to change the message only